mantaray/config.py

```diff
@@ -245,6 +230,7 @@ def __init__(
         self._server_entry.var.set(initial_config["host"])
         self._ssl_var.set(initial_config["ssl"])  # must be before port
         self._port_entry.var.set(str(initial_config["port"]))
+        self._audio_var.set(initial_config["audio_notification"])
```

Comment:
Nit: While a separate `.set()` works, it would be simpler to create `_audio_var` with `value=initial_config["audio_notification"]`.